projects/knot-dns/build.sh

```diff
@@ -46,6 +46,7 @@ bash .bootstrap
 
 cd $SRC/gnutls
 touch .submodule.stamp
+apt-get install -y gettext
```

Comment:
Please move this to DockerFile as part of other install packages. So that this is run once qand not on every build trigger (sanitizer-fuzzing engine config)